client/my-sites/site-settings/reader-settings/index.tsx

```diff
@@ -1,32 +1,21 @@
 import { Card } from '@automattic/components';
 import { localizeUrl } from '@automattic/i18n-utils';
-import { ToggleControl } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
 import FormLegend from 'calypso/components/forms/form-legend';
 import SupportInfo from 'calypso/components/support-info';
 import SettingsSectionHeader from 'calypso/my-sites/site-settings/settings-section-header';
 
-type Fields = {
-	wpcom_reader_views_enabled?: boolean;
-};
-
 type ReaderSettingsSectionProps = {
-	fields: Fields;
-	handleAutosavingToggle: ( field: string ) => () => void;
 	isAtomic: boolean | null;
-	isRequestingSettings: boolean;
-	isSavingSettings: boolean;
 	siteIsJetpack: boolean | null;
 };
 
-const ReaderSettingsSection = ( {
-	fields,
-	handleAutosavingToggle,
-	isRequestingSettings,
-	isSavingSettings,
-	isAtomic,
-	siteIsJetpack,
-}: ReaderSettingsSectionProps ) => {
+// Note this component is not in use as of https://github.com/Automattic/wp-calypso/pull/84828. This
```

Comment:
I left the definition and file structure in place for visibility and easy access in the future when we need to re-add settings.

Im still uncertain if I should just delete the file instead. Curious if other teammates have thoughts @eoigal @roo2

Reply:
I think we should just delete it. If we need to resurrect it we can find it from past PRs.

Reply:
Yeah that seems totally fair to me. Thank you 😁